### ISSUE TITLE
Failed to view Experiments with specified logDir

### ISSUE THREAD

**Issue by EternalConfession** [microsoft/nni#3520]
**Environment**:
- NNI version: 2.1
- NNI mode (local|remote|pai): local
- Client OS: Ubuntu 
- Server OS (for remote mode only): N/A
- Python version: 3.7
- PyTorch/TensorFlow version: PyTorch 1.6
- Is conda/virtualenv/venv used?: Yes
- Is running in Docker?: No

**Log message**:
 - nnimanager.log: 
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**What issue meet, what's expected?**:
I changed the logDir in Experiment Config.yml as:
```yml
logDir: /data/Exps
```
After training, when I ran the following code
```bash
nnictl view [exp_id]
```

I met the follow error:

```bash
INFO:   Stderr:
/home/jianhui/anaconda3/lib/python3.7/site-packages/pyparsing.py:3174: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
-----------------------------------------------------------------------
                Experiment start time 2021-04-01 12:57:00
-----------------------------------------------------------------------
/home/jianhui/anaconda3/lib/python3.7/site-packages/pyparsing.py:3174: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
/home/jianhui/anaconda3/lib/python3.7/site-packages/json_tricks/nonp.py:225: JsonTricksDeprecation: `json_tricks.load(s)` stripped some comments, but `ignore_comments` was not passed; in the next major release, the behaviour when `ignore_comments` is not passed will change; it is recommended to explicitly pass `ignore_comments=True` if you want to strip comments; see https://github.com/mverleg/pyjson_tricks/issues/74
  JsonTricksDeprecation)
/home/jianhui/anaconda3/lib/python3.7/site-packages/pyparsing.py:3174: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
-----------------------------------------------------------------------
                Experiment start time 2021-04-08 18:29:18
-----------------------------------------------------------------------
Failed to create log dir: AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

  assert(fs.existsSync(dbDir))

    at SqlDB.init (/home/jianhui/anaconda3/lib/python3.7/site-packages/nni_node/core/sqlDatabase.js:72:9)
    at NNIDataStore.init (/home/jianhui/anaconda3/lib/python3.7/site-packages/nni_node/core/nniDataStore.js:35:21)
    at initContainer (/home/jianhui/anaconda3/lib/python3.7/site-packages/nni_node/main.js:87:14)
    at utils_1.mkDirP.then (/home/jianhui/anaconda3/lib/python3.7/site-packages/nni_node/main.js:146:15)
```

I also checker ~/nni_experiments/.experiment

The logDir logged in the metadata are correct.

**How to reproduce it?**: 

**Additional information**:

**Comment by EternalConfession** [comment]
It seems that the nnictl view comman will change the logDir.

I first checked the .experiment, the logDir was right.

After I ran:
```bash
 nnictl view [exp_id]
```

The logDir in ~/nni-experiments/.experiment was rewritten by this command.

**Comment by J-shang** [comment]
Hello @EternalConfession , thanks for your contribution, we found it is a bug and fixed it in pr #3545 and will release in nni v2.2.

**Comment by kvartet** [comment]
hello @J-shang, I met this problem too.
My nni version is v2.2, and when running [Retiarii mnist example](https://github.com/microsoft/nni/blob/master/examples/nas/multi-trial/mnist/search.py), the trail failed with the error message:
``
{"error":"ENOENT: no such file or directory, open '3p715gyd/trials/yK4FG/stderr'"}
``

When I set the ``experiment_working_directory`` to a relative path, it fails again, but to an absolute path or use nni v2.1, it works.

**Comment by J-shang** [comment]
@kvartet  fix in pr #3607
